### ISSUE TITLE
Feature Request: Execute PS after Run Profile completes

### ISSUE THREAD

**Issue by microfoundry** [lithnet/miis-autosync#164]
Case: Utilizing SQL MA with Delta Table and would like the ability to execute PS script to truncate table after DI run has completed...

Thank you for the consideration,

Terry

**Comment by ryannewington** [comment]
Hi @microfoundry

Check out https://github.com/lithnet/miis-autosync/wiki/Execution-controller-scripts 

You can use these to do what your are after.

Ryan

**Comment by microfoundry** [comment]
Doah - RTFM! Thank you Ryan, that was exactly what I needed.

But as a follow on: I was trying to run the Invoke-Sqlcmd cmdlet when I got the following error:

Mixed mode assembly is built against version ‘v2.0.50727’ of the runtime and cannot be loaded in the 4.0 runtime without additional configuration information.

Using some Bing-Foo and a wild guess, I modified the Lithnet.Miisserver.AutoSync.exe.config file to look like:

`<startup useLegacyV2RuntimeActivationPolicy="true">`
`       <supportedRuntime ver.... />`
`</startup>`

Might be worth mentioning in the Wiki section?

Thank you again,

Terry
